build/prerequisites.ps1

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+$ErrorActionPreference = 'Stop'
+& "$PSScriptRoot/set-env.ps1"
+
+Push-Location (Join-Path $PSScriptRoot "../src/Simulation/CsharpGeneration")
+    .\FindNuspecReferences.ps1
+Pop-Location
+
+Push-Location (Join-Path $PSScriptRoot "../src/Simulation/Simulators")
+    .\FindNuspecReferences.ps1
+Pop-Location
+
+# Native Quantum Simulator pre-reqs
+if ($Env:ENABLE_NATIVE -ne "false") {
+    Push-Location (Join-Path $PSScriptRoot "../src/Simulation/Native")
+        .\install-prereqs.ps1
+    Pop-Location
+} else {
+    Write-Host "Skipping installing prerequisites for native simulator because ENABLE_NATIVE variable set to: $Env:ENABLE_NATIVE"
+}
+
+# At the moment the QIR Runtime build isn't enabled locally by default.
+if ($Env:ENABLE_QIRRUNTIME -eq "true") {
```

Comment:
In the e2e build this is also enabled (and we said we didn't want it enabled on e2e yet). The problem is that the `set-env.ps1` sets it automatically when running on a build agent (regardless of the build).
My recommendation is to not set it automatically on the set-env.ps1 script. Instead, create a `Enable.QIR` variable in the ci.yml and set it to true there.

Reply:
Done. Setting the variable in ci.yml is exactly what I need. Thanks!